### PR TITLE
Update to latest mock-mesos-server

### PR DESCRIPTION
**Pull request by philipnrmn**
Playing around with mock-mesos-server to stop flakiness in telegraf tests.

### CODE REVIEW

**Comment by philipnrmn** [comment]
Well, it fixes the contextDeadlineExceeded error (https://jira.mesosphere.com/browse/DCOS-56469) but now the inputs.mesos test is failing reliably with missing measurements

**Comment by philipnrmn** [comment]
I did see another contextDeadlineExceeded flake, so I bumped the test timeout in both places just to be sure.

**Comment by philipnrmn** [comment]
Three passing builds in a row :relaxed:

File: plugins/inputs/dcos_containers/mock_mesos_server.go [hunk 1]
```diff
@@ -15,46 +10,60 @@ import (
...
+	GET_TASKS      = []byte{8, 13}
 )
 
 // startTestServer starts a server and serves the specified fixture's content
 // at /api/v1
 func startTestServer(t *testing.T, fixture string) *httptest.Server {
 	router := http.NewServeMux()
+	containers, cOK := loadFixture(t, filepath.Join(fixture, "containers.bin"))
+	state, sOK := loadFixture(t, filepath.Join(fixture, "state.bin"))
+	tasks, tOK := loadFixture(t, filepath.Join(fixture, "tasks.bin"))
```
Comment (branden) [inline plugins/inputs/dcos_containers/mock_mesos_server.go:32]: Nice, moving the disk reads out of the request handler should be a big help 👍

**Review by branden — APPROVED** [review]
That green check mark 😍

**Review by gracedo — APPROVED** [review]

**Pull request merged**